docs/docs/cookbook/cel.md

```diff
@@ -12,3 +12,10 @@ before the provided date. This is particularly useful when attached to a
 target.signing_time >= timestamp('2025-05-31T00:00:00Z')
 ```
 
+## Prevent users from disabling gatekeeper
+
+Create a signing ID rule for `platform:com.apple.spctl` and attach the following CEL program
+
+```clike
+['--global-disable', '--disable', '--remove'].exists(flag, flag in args) ? BLOCKLIST : ALLOWLIST
```

Comment:
If we're handling --remove you may also want to add --add, otherwise a user could add a rule for a program on disk that would otherwise be denied.